packages/commands/src/index.ts

```diff
@@ -537,14 +537,36 @@ export class CommandRegistry {
       event
     );
 
-    // If there is no exact match and no partial match, replay
-    // any suppressed events and clear the pending state.
-    if (!exact && !partial) {
-      this._replayKeydownEvents();
```

Comment:
This should not be removed, it seems that we cannot use the keyboard keys (other than the Lumino shortcut) anymore.
On my side I can't write in cell, I can't use web browser shortcuts...)

Reply:
I have added it back